### ISSUE TITLE
option to display party frames outside of arena

### ISSUE THREAD

**Issue by Gimlehr** [vendethiel/GladiusEx#32]
As per title would be nice to have the option to show party frames all the time not just when entering an arena

**Comment by vendethiel** [comment]
`/gex test5` is what I do.
I have two profiles, one for pve without arena frames displayed, one for pve.

**Comment by Gimlehr** [comment]
Ok that's probably workable for now its mostly when out of arena between games being able to interact with the party frames to remove someone or inspect them when running pug's, I currently need to have blizz party frames showing for that.

**Comment by vendethiel** [comment]
you can enable the Clicks module in party so left click targets.

**Comment by Gimlehr** [comment]
Clicking itself is not the issue its the fact party frames only show in arena unless you do a /getx test2-5

Ideally there would be an option to have party frame show all the time so they could replace blizz party frames and just have arena frames show/hide when entering an arena match.

**Comment by Gimlehr** [comment]
I may not be explaining myself particularly well, I'll mock up some screenshots if that helps

**Comment by vendethiel** [comment]
No no, I think I understand, there's just no way to do that ATM because the addon has a few bugs when you use it that way: it was never considered that a party member might be replacing mid-instance for example.

**Comment by Gimlehr** [comment]
Ok fair I tend to run a lot of pugs rather than a fixed team unfortunately.

We often have to go though a number of players before finding someone that fits.

**Comment by Gimlehr** [comment]
Maybe something to consider for future as the party frames are better than most and would make a good replacement for othe party frames.

Splitting the functions that show/hide frames to allow for showing/hiding individually may be an option, I only had a very quick look through the code and my lua knowledge isn't amazing.

**Comment by vendethiel** [comment]
> showing/hiding individually

What do you mean with "individually"? you mean to have a solo one or a 4-people group one? because otherwise, "2v2" "3v3" "5v5" do the job.

**Comment by Gimlehr** [comment]
By individually I mean party and arena. looks like the code passes 'unit' into the show/hide function and arena/party frames are dealt with together.

**Comment by Gimlehr** [comment]
Technically a 1-5 for party would be needed to enable party frames to be used as replacements for blizz party frames, for arena 2, 3 and 5 is all that's needed I agree

**Comment by vendethiel** [comment]
You can alreaady do that, though?
<img width="681" alt="image" src="https://user-images.githubusercontent.com/199499/106271760-4c1f3d00-6230-11eb-87f1-27b74cb75595.png">

Maybe I missed something

**Comment by Gimlehr** [comment]
Thats true I think party frame is fine when you're in a party in arena it was more around if future implementation enabled displaying party frames outside of arena being ae to handle 1-5 players in a party helps when building a group, its just the event that triggers displaying party frames is arena entry or test as far as I can tell

**Comment by Gimlehr** [comment]
If I have party enabled /gex 5 shows my party + placeholders for unfilled spots

**Comment by vendethiel** [comment]
Ah, that's what you mean. Yeah that's a bit annoying, but I tend not to use it when forming the group so I didn't really run into that issue.
